app/controllers/application_controller.rb

```diff
@@ -80,7 +80,7 @@ def require_user
       unless current_user
         store_location
         flash[:notice] = "You must be logged in to access this page"
-        redirect_to login_url+'?return_to=' + URI.encode(request.env['PATH_INFO'])
```

Comment:
In the future, perhaps smaller, independent changes could be submitted in separate pull requests, however, where possible. Thanks, though, this is a good catch and a welcome contribution!